Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:10 AS builder
+RUN dpkg --add-architecture i386 && \
+	apt-get update -y && \
+	apt-get install -y --no-install-recommends \
+		build-essential=12.6 \
+		zlib1g:i386=1:1.2.11.dfsg-1+deb10u2 \
+		default-libmysqlclient-dev:i386=1.0.5 \
+		gcc-multilib=4:8.3.0-1 \
+		cmake=3.13.4-1 && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/* && \
+    mkdir -p /opt/var/astonia/zones/generic && \
+    mkdir -p /opt/var/astonia/.obj
```

Comment:
I saw that you included a directory ".obj" with a .gitkeep file in it. This has proliferated to this line here. I'm wondering if it would make more sense to push the ".obj" creation to the makefile using `mkdir -p`